### DISCUSSION TITLE
Repeating Fields

### DISCUSSION THREAD

**Discussion by finidev** [elementor/elementor#25237] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I find myself often stuck when it comes to projects needing to use repeating fields via ACF. I would love for elemental to have this natively, so I do not have to depend on third-party plug-ins, such as dynamic.ooo as this plugin is very resource heavy.

### Describe the solution you'd like.

I would simply love to have the ability to natively use element, or only to solve the problem for dynamic repeating fields.

### Describe alternatives you've considered.

As mentioned above, I often find myself using dynamic.ooo in order to solve these problems.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by theavic** [comment]
This has been a feature which has been requested repeatedly, and while I certainly hope that they support it soon - I don't know if it's a priority
